lib/utils/dataflow/src/index.js

```diff
@@ -380,9 +432,10 @@ const mapper = (mapfn, opt) => {
 
   // Map an input doc to an output doc, store both the input and output and
   // pass the output to the configured sink service
-  /* eslint complexity: [1, 6] */
+  /* eslint complexity: [1, 10] */
```

Comment:
Looks like a workaround to make eslint happy, wouldn't it be better to actually lower the complexity by splitting the code into multiple functions?